bin/auditwheel

```diff
@@ -18,12 +18,16 @@ libs = [
     "libcublas.so",
     "libcublas.so.11",
     "libcublas.so.12",
+    "libcusolver.so.11",
```

Comment:
Do we want to remove `*.so.12` as well?

Reply:
I didn't see that in the generated wheels, but it's a good idea to exclude them both to preserve the compatibility with next versions. Thanks!
```suggestion
    "libcusolver.so.11",
    "libcusolver.so.12",
```